### PR TITLE
Adjusts cook's culinary school techniques to be tied to wearing the uniform

### PR DESCRIPTION
**Pull request by vuonojenmustaturska**
follow-up to #35417 based on Kor's feedback, I was going to commit this into that branch but it got merged. Oh well.

:cl: Naksu
balance: Cook's CQC is now conditional to wearing the uniform
/:cl:

### CODE REVIEW

File: code/modules/clothing/suits/jobs.dm [hunk 1]
```diff
@@ -76,6 +76,27 @@
 	allowed = list(/obj/item/kitchen)
 	togglename = "sleeves"
 
+/obj/item/clothing/suit/toggle/chef/equipped(mob/user, slot)
+	. = ..()
+	if(!ishuman(user))
+		return
+	if(slot == slot_wear_suit)
+		var/mob/living/carbon/human/H = user
+		if (istype(H.get_item_by_slot(slot_head), /obj/item/clothing/head/chefhat))
```
Comment (KorPhaeron) [inline code/modules/clothing/suits/jobs.dm]: Does this include the cooks cap?

**Comment by ChuckTheSheep** [comment]
Make it area based instead, he has to defend his kitchen from siege.

File: code/modules/clothing/suits/jobs.dm (hunk 1 above)
Comment (vuonojenmustaturska) [reply code/modules/clothing/suits/jobs.dm]: If you mean the white cap, no. The white cap is a generic item and not tied to the job.

Comment (vuonojenmustaturska) [reply code/modules/clothing/suits/jobs.dm]: also, 
image this is not a cap

**Comment by Xhuis** [comment]
I agree that it should probably be tied to area. With always-on CQC, I could easily foresee people picking chef and immediately starting to tide with their newfound unarmed power, but if it's restricted to the Kitchen they at least have to *try* to cook.

**Comment by Qustinnus** [comment]
and if you make it area based maybe put the boolean for "is_in_department" on a high enough level so that i can check it for any job :^)


if thats not too much work ofc

**Comment by DaxDupont** [comment]
Remove hat requirement since the cook hats look like shit

**Comment by Xhuis** [comment]
You should also keep in mind that plasman cooks are incapable of having CQC with this PR, so some sort of workaround for it might be a good idea. I don't know what that might be, though...

**Comment by KorPhaeron** [comment]
Chef needs to charge his CQC powers by making food or something I dunno

I guess area might be easier

**Comment by DaxDupont** [comment]
Make sure to include the bar, since most maps have the kitchen leading out into the bar area, so if someone is outside smashing your tables with a spear you can still ass blast them

**Comment by darkpaladin109** [comment]
Yeah, just make it tied to their work area. CQC's stun ability is going to be cancer if it's tied to just the uniform, since having it on your person is not exactly a hard task.

File: code/modules/clothing/suits/jobs.dm (hunk 1 above)
Comment (francinum) [reply code/modules/clothing/suits/jobs.dm]: There also isn't a jobcheck here, unlike for the hat equip.

**Comment by optimumtact** [comment]
closed on request

**Pull request closed without merging**